modules/matrix_operations/include/matrix_operations.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Kiseleva Anastasia
+
+#ifndef MODULES_MATRIX_OPERATIONS_INCLUDE_MATRIX_OPERATIONS_H_
+#define MODULES_MATRIX_OPERATIONS_INCLUDE_MATRIX_OPERATIONS_H_
+
+#include<vector>
+#include<cmath>
+
+class Matrix {
+ private:
+    unsigned int rows;
+
+    unsigned int cols;
+
+    std::vector<std::vector<double>> data;
+
+
+ public:
+    Matrix(unsigned int row, unsigned int col);
+
+    Matrix(const Matrix& tmp);
+
+    ~Matrix();
+
+    void set_data(std::vector < std::vector < double>> new_data);
```

Comment:
```suggestion
    void set_data(std::vector<std::vector<double>> new_data);
```
Кодстайл разрешает запись без пробелов, и без них метод будет выглядеть приятнее.